packages/headless-styles/package.json

```
@@ -33,7 +33,10 @@
   "files": [
     "npm/**/*"
   ],
-  "sideEffects": false,
+  "sideEffects": [
+    "**/*.scss",
```

Shouldn't this be the transformed CSS (i.e. `.css`)? I've never seen putting scss files in sideEffect. Literally, ever?

```suggestion
    "**/*.css",
```

---

This instructs the bundler (rollup, currently) to include imported files that are not directly used in the code.  In this case `import 'buttonCSS.scss'` would be ignored and not processed if we didn't have it.  After we move off of rollup we will probably be able to remove the scss and add css.